docs/client_client-node.js.html

```
@@ -38,32 +38,40 @@ <h1 class="page-title">client/client-node.js</h1>
 const Deferred = require('../lib/deferred');
 
 
 /**
  * Polyfill http request for node
  */
 Client.fetchResponseUrl = function(url, timeout = 3000) {
+    let req;
+
     return new Deferred({
         timeout,
+        onExpire: () => req &amp;&amp; req.destroy(),
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)